### PR TITLE
Release 1.0.1

### PR DESCRIPTION
**Pull request by alexshoe**
- Update to version 1.0.1 for the recently merged changes
- Updates `CHANGELOG` to reflect changes

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,6 +15,8 @@
   "maintainer": "Alex Hsu <alex.hsu@plot.ly>",
   "license": "MIT",
   "dependencies": {
+    "16": "0.0.2",
+    "npm": "^8.15.0",
```
Comment (alexcjohnson) [inline package.json:19]: What's 16? Anyway I wouldn't list npm/node in dependencies, but if you want to list their recommended versions in `CONTRIBUTING.md` that could be worthwhile.

Comment (alexshoe) [reply package.json]: Not sure how that got there. I moved removed dependencies from `package.json` and listed recommended versions in `CONTRIBUTING.md`

File: CHANGELOG.md [hunk 2]
```diff
@@ -5,6 +5,15 @@ This project adheres to [Semantic Versioning](https://semver.org/).
 
 ---
 
+## [1.0.2] - 2022-10-20
```
Comment (alexcjohnson) [inline CHANGELOG.md:8]:
```suggestion
## [1.0.1] - 2022-10-20
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -5,6 +5,15 @@ This project adheres to [Semantic Versioning](https://semver.org/).
...
+
+## Changed
+
+- [#53](https://github.com/plotly/dash-player/pull/53) Use README as package description
+- [#54](https://github.com/plotly/dash-player/pull/54) Bump `npm` and `node` versions
```
Comment (alexcjohnson) [inline CHANGELOG.md:13]:
```suggestion
- [#54](https://github.com/plotly/dash-player/pull/54) Bump `npm` and `node` versions and remove an obsolete file to reduce the package size
```

**Review by alexcjohnson — APPROVED** [review]
💃 Let's do it! (after one last minor comment https://github.com/plotly/dash-player/pull/55#discussion_r1001178761 )

**Pull request merged**